demos/github-demo/src/isograph-components/UserRoute.tsx

```diff
@@ -1,17 +1,17 @@
-import React from 'react';
-import { iso, useLazyReference, useRead } from '@isograph/react';
-import { Container } from '@mui/material';
+import React from "react";
```

Comment:
if you do `Preferences: Open Workspace Settings (JSON)` and add `  "prettier.configPath": ".prettier.config.js"`, this formatting issue should be fixed.

I think VSCode doesn't like the fact that we're not using prettierrc. It's something I'll look into. But in the mean time, that seems to fix it. Or run `pnpm run format`

Reply:
Fixed with [`8abe8a5` (#12)](https://github.com/isographlabs/isograph/pull/12/commits/8abe8a5bc56b06b5154d46ea4253b1952627cc8c)